src/views/Dashboard.vue

```diff
@@ -0,0 +1,29 @@
+<template lang="pug">
+  .dashboard-container
+    nav-bar
+</template>
+
+<script>
+import NavBar from './NavBar'
+
+export default {
+  name: 'dashboard',
+  components: {
+    NavBar
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+@import '@/scss/commons/colors.scss';
```

Comment:
Does the `@` work? I think it's just a shorthand for JS, not SCSS :thinking: . BTW, no need for the extension

Reply:
If I delete the `@`, it doesn't work. Same goes for the extension. Both fail in the compilation :thinking: